src/assets/scss/checkbox.scss

```diff
@@ -1,20 +1,26 @@
-fieldset {
-    border: none;
-    display: flex;
-    flex-direction: column;
-    margin: 20px;
-    align-items: flex-start;
-    min-width: 400px;
+.container-checkbox-component{
+    &.checkbox-group-filled {
+        border: none;
+        display: flex;
+        flex-direction: column;
+        margin: 20px;
+        align-items: flex-start;
+        min-width: 400px;
+    }
 }
 
-label {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    color: $other-dark;
+.filled {
+    
+    &label{
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        color: $other-dark;
+    }
+    
 }
 
-.input-box, .filled {
+.container-checkbox-component, .filled {
```

Comment:
It's some reset default styles, why don't put it in early lines? At this point you already have establish some styles to these same classes.

Reply:
On the site i picked up this checkbox was that way so i left it this way, i don't know what happens if i change.